src/App.js

```diff
@@ -1,5 +1,26 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import InvitationForm from './Invitation/InvitationForm';
+import Testing from './Invitation/Testing';
+import RSVP from './Invitation/RSVP';
 
 export default function App() {
-  return <h1>Hello world!</h1>;
+  console.log('Pls do not use /testing as the path name for your route');
+  /**
+   * 1. npm run server:dev
+   * 2. npm run dev:w (if npm run dev is not working)
+   * 3. Add the route path after localhost:3000
+   *
+   */
+  return (
+    <Router>
+      <Routes>
+        <Route path='/' element={<h2>Just localhost</h2>} />
+        <Route path='/sometest' element={<Testing />} />
+        <Route path='/rsvp' element={<RSVP />} />
```

Comment:
Also small, but could we be a little bit more descriptive with page titles, RSVP could refer to any of the pages on the RSVP section, could be RSVP-gettingStarted or RSVP-preview etc

Reply:
Hey Dylan, thanks for pointing out. I just made the update. These naming are mainly for testing purpose, all future naming will be more descriptive. 😊